rebasehelper/application.py

```diff
@@ -142,8 +142,7 @@ def _prepare_spec_objects(self):
             logger.debug("argument passed as a new source is a version")
             version, extra_version = SpecFile.split_version_string(self.conf.sources)
             self.rebase_spec_file.set_version(version)
-            if extra_version:
-                self.rebase_spec_file.set_extra_version(extra_version)
+            self.rebase_spec_file.set_extra_version(extra_version)
```

Comment:
I guess, that it was correct.
What about in case if extra_version doesn't exist?
Will it work?